### PR TITLE
Installs and configure Devise

### PR DESCRIPTION
**Pull request by Drocheg**
-Creates users with the default columns created by Devise
-Adds first_name and last_name columns to users

### CODE REVIEW

**Review by Gmarn — CHANGES_REQUESTED** [review]
Now you have travis running I will like it before start correction on that. So please do me a rebase and when build passed I look on the PR ;)

File: db/migrate/20180815132933_add_name_to_user.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddNameToUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :first_name, :string
```
Comment (Gmarn) [inline db/migrate/20180815132933_add_name_to_user.rb]: It is seems you forgot `first_name` is a required columns.

File: db/migrate/20180815132933_add_name_to_user.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    add_column :users, :last_name, :string
```
Comment (Gmarn) [inline db/migrate/20180815132933_add_name_to_user.rb]: same for `last_name`

**Review by Gmarn — DISMISSED** [review]

File: db/migrate/20180815180513_add_required_to_user_name.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class AddRequiredToUserName < ActiveRecord::Migration[5.1]
+  def change
+    change_column_null :users, :first_name, false
```
Comment (leoncalermo) [inline db/migrate/20180815180513_add_required_to_user_name.rb]: I like this approach, but...  knowing that this the migration where you are creating this column is in this same PR i would recommend to update the migration where you  added them for the first time. the `AddNameToUser` one. you need to make a rollback to that migration, then update it, and then run it again.

**Review by leoncalermo — CHANGES_REQUESTED** [review]
just that small change and we are ready to go!

**Review by leoncalermo — APPROVED** [review]
todo bien

**Pull request merged**